src/jobs/microk8s_install.yml

```diff
@@ -63,18 +82,21 @@ steps:
         # install microk8s and selected addons
         sudo snap install microk8s --classic --channel << parameters.microk8s_channel >>
         export PATH=$PATH:/snap/bin
-        sudo microk8s.start && sudo microk8s.status --wait-ready
-        sudo microk8s.enable ingress dns storage && sudo microk8s.status --wait-ready
+        sudo microk8s.start && sudo microk8s_wait_ready 20
+
+        # enabling plugins one at a time to prevent a race condition
+        # see https://github.com/ubuntu/microk8s/issues/706
+        sudo microk8s.enable dns && microk8s_wait_ready 20
```

Comment:
all those commands in a single block start to scares me ...

Reply:
What do you propose instead? I'm open to suggestions.